app/server/dao/src/main/java/io/syndesis/server/dao/manager/DataManager.java

```diff
@@ -182,6 +183,20 @@ private void loadData() {
         }
     }
 
+    private static Map<String, String> merge(Map<String, String> existingProps, Map<String, String> newProps) {
+        Map<String, String> mergedProps = new HashMap<>();
+        mergedProps.putAll(existingProps);
+        // New properties will override any existing property with same name
+        mergedProps.putAll(newProps);
+        LOGGER.info("Merging {} new configured properties with {} existing configured properties: {} merged configured properties",
```

Comment:
Not sure if this merits an INFO level log

Reply:
I could lower to `DEBUG`. However I feel like it is something we should notify to a user: in my case, if the info was present, I had not to go deep in code to understand that there was a merge of properties that was the root cause of the failure.